pallets/astar-xcm-benchmarks/src/generic/benchmarking.rs

```diff
@@ -43,9 +43,9 @@ mod benchmarks {
         let fee_asset = Location::parent();
 
         let instruction = Instruction::<XcmCallOf<T>>::BuyExecution {
-            fees: (fee_asset, u128::MAX).into(), // should be something inside of holding
-            // this should not be Unlimited, as xcm-executor will skip buying the
-            // exceution altogether.
+            // Should be something inside of holding register.
+            fees: (fee_asset, 1_000_000_000_000_000_000u128).into(),
```

Comment:
Not sure what you're referring to?

The benchmark is essentially the same, I've only put a smaller value as the amount since the holding register won't contain this with my recent changes. Shouldn't impact execution time though.

Reply:
yes correct, it's just a value update, my bad